lib/liquid/combinators/tags/for.ex

```diff
@@ -51,6 +51,22 @@ defmodule Liquid.Combinators.Tags.For do
   alias Liquid.Combinators.{General, Tag}
   alias Liquid.Combinators.Tags.Generic
 
+  @type t :: [
```

Comment:
Shouldn't a For type be just a tagged tuple instead of a list of tagged tuples?

Reply:
Every tag is a keyword list (which is a list of tagged tuples)